analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -69,6 +69,14 @@ protected override void Initialize(SonarAnalysisContext context)
             SyntaxKind.OperatorDeclaration,
             SyntaxKind.MethodDeclaration);
 
+        context.RegisterNodeAction(
+            c => Analyze<SyntaxNode>(context, c, x =>
+            {
+                var localFunction = (LocalFunctionStatementSyntaxWrapper)x;
+                return (SyntaxNode)localFunction.Body ?? localFunction.ExpressionBody;
```

Comment:
Both, `Body` and `ExpressionBody` will  need to be tested in the runner test above